example/pythonlib/basic/1-simple/qux/test/src/test_dummy.py

```diff
@@ -0,0 +1,17 @@
+import unittest
```

Comment:
This file should exercise some of the logic in the associated `PythonModule` rather than being entirely standalone